### PR TITLE
Added required milestone

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added required milestone

### CODE REVIEW

File: docs/Using-EthSigner/Azure.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ option set to `8590` to avoid conflict with the default EthSigner listening port
     besu --network=dev --miner-enabled --miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-http-cors-origins="all" --host-whitelist=* --rpc-http-enabled --rpc-http-port=8590 --data-path=/tmp/tmpDatdir
     ```
 
+!!! important 
+    To use EthSigner for signing transactions, the Ethereum client genesis file must include at least the
```
Comment (rain-on) [inline docs/Using-EthSigner/Azure.md]: My version?
EthSigner _requires_ a chainId to be used when signing transactions, as such, it is only capable of submitting transactions to a blockchain which is in (or beyond) the Byzantium milestone (i.e. chainId is used for replay prevention). [Is it byzantium?!]

Comment (rain-on) [reply docs/Using-EthSigner/Azure.md]: "which has passed the byzantium milestone" maybe?

Comment (rain-on) [reply docs/Using-EthSigner/Azure.md]: Think its actually the SpuriousDragon milestone (not Byzantium <-- is way too new)

File: docs/Using-EthSigner/Azure.md [hunk 2]
```diff
@@ -31,8 +31,9 @@ option set to `8590` to avoid conflict with the default EthSigner listening port
     ```
 
 !!! important 
-    To use EthSigner for signing transactions, the Ethereum client genesis file must include at least the 
-    `byzantium` milestone.
+    EthSigner requires a [chain ID](https://besu.hyperledger.org/en/stable/Concepts/NetworkID-And-ChainID/) to be 
+    used when signing transactions. The Ethereum client genesis file must include at least the Spurious Dragon
```
Comment (rain-on) [inline docs/Using-EthSigner/Azure.md]: Yeah - I know this is largely a copy of the previously accepted text ... but ...

.. used when signing transactions. Thus the downstream Ethereum client's chain must be operating in a milestone which supports "replay protection" - i.e. SpuriousDragon (defined as eip158 in the genesis file) or later.

Comment (MadelineMurray) [reply docs/Using-EthSigner/Azure.md]: Fixed

**Pull request closed without merging**